### PR TITLE
Added YouTube tutorial to contribution guide

### PR DESCRIPTION
**Pull request by RyannDaGreat**
I've added a YouTube tutorial addressing https://github.com/Anarios/return-youtube-dislike/issues/510
 
It walks the user through the cloning process, running NPM, and installing the Chrome extension. It's newbie-friendly, so it should encourage more people to contribute!

The video can be found here: https://www.youtube.com/watch?v=mdOj6HYE3_0

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Welcome To Return YouTube Dislikes contributing guide
+x# Welcome To Return YouTube Dislikes contributing guide
```
Comment (krystian3w) [inline CONTRIBUTING.md:1]: what is that

**Review by krystian3w — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (RyannDaGreat) [reply CONTRIBUTING.md]: Oh my! That's a typo lol...thank you!

**Review by krystian3w — APPROVED** [review]

**Comment by JacksonChen666** [comment]
this pr, i would assume, resolves #510 

(using specific keywords and mentioning an issue will close the respective issue when the pr is merged)

**Comment by sy-b** [comment]
Just a test

fixes #514

**Comment by cyrildtm** [comment]
> (probably)

[yes](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)

**Review by ErykDarnowski — APPROVED** [review]

**Pull request merged**